### PR TITLE
Move styles from global to local

### PR DESCRIPTION
**Pull request by rojasAndresTapia**

### CODE REVIEW

File: src/components/CardComponent/CardComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export const CardComponent = ({
+  description,
+  primaryLogo,
+  primaryLogoAltText,
+  optionalText,
+  secondaryLogo,
+  secondayLogoAltText,
+  buttonText,
+  children,
+}: ICard) => {
```
Comment (manudous) [inline src/components/CardComponent/CardComponent.tsx]: export const CardComponent:React.FC<Props> = props => {
const {discription,....} = props
}

File: src/components/CardComponent/CardComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Card } from '@material-ui/core';
+
+import * as classes from './CardComponentStyles';
+
+export interface ICard {
```
Comment (manudous) [inline src/components/CardComponent/CardComponent.tsx]: Rename ICard to Props

File: src/pods/home/home.container.tsx [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
-      </div>
-      <div className={classes.containerTitle2}>
-        <h3 className={classes.title(174)}>a formato europass</h3>
-      </div>
-      <img src={LogoEuropass} alt="Logo de Europass" />
-      <div className={classes.buttonContainer}>
-        <button className={classes.beginButton}>Comenzar</button>
-      </div>
-    </main>
+    <CardComponent
```
Comment (manudous) [inline src/pods/home/home.container.tsx]: <CardComponent {props} />

secondayLogoAltText='Logo de Europass'

File: src/common/components/card/card.component.styles.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  height: 23px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 19px;
+  line-height: 23px;
+  padding: 0;
+  margin: 0;
+`;
+
+export const beginButton = css`
```
Comment (manudous) [inline src/common/components/card/card.component.styles.ts]: are u using this style?

File: src/common/components/card/card.component.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Card as CardUI } from '@material-ui/core';
```
Comment (manudous) [inline src/common/components/card/card.component.tsx]: rename better to CardMui

Comment (rojasAndresTapia) [reply src/common/components/card/card.component.tsx]: Done

File: src/common/components/card/card.component.styles.ts (hunk 4 above)
Comment (rojasAndresTapia) [reply src/common/components/card/card.component.styles.ts]: Done

**Pull request merged**